iceoryx_posh/test/moduletests/test_roudi_mempool_introspection.cpp

```diff
@@ -21,14 +21,8 @@
 using namespace ::testing;
 using ::testing::Return;
 
-#define private public
-#define protected public
-
```

Comment:
:partying_face:

Reply:
only 5 remaining. Those also need to go the way of the dodo in order to support Windows.